README.md

```diff
@@ -34,6 +34,15 @@ The Mapbox Navigation SDK is also available [for Android](https://github.com/map
 
 To install Mapbox Navigation using [CocoaPods](https://cocoapods.org/):
 
+1. Create a Mapbox API token (from your account page on [mapbox.com](https://www.mapbox.com)) with the      `DOWNLOADS:READ` scope. 
+   **PLEASE NOTE: This is not the same as your production Mapbox API token.** 
+   Once you have the token, create a `~/.netrc` which looks like this:
+   ```
+   machine api.mapbox.com 
+      login mapbox
+      password <INSERT API TOKEN>
+   ```
+
```

Comment:
This is the section about installing the latest stable release, v0.40.0. Can you move these three changes down to the section about installing the beta?

We’ll remove this section once we publish a final release of v1.0.0. In the meantime, by the time map SDK v5.10.0 goes out, we’ll have to publish a v0.40.1 that prevents CocoaPods and Carthage from automatically upgrading to v5.10.0: #2447.